src/bio/bio_internal.h

```diff
@@ -72,29 +72,45 @@ enum bio_bs_state {
 	BIO_BS_STATE_REINT
 };
 
+/*
+ * SPDK device health monitoring.
+ */
+struct bio_health_monitoring {
```

Comment:
We are just monitoring device health here correct? If so, can we call this bio_dev_health just to keep data structure names shorter?